build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -225,5 +225,12 @@
           DestinationFiles="@(FilesToCopy->'$(StageSymbolsDirectory)\%(RecursiveDir)%(Filename)%(Extension)')" />
 
       <Delete Files="@(PdbsToClean)" />
+
+      <!-- workaround for https://github.com/Microsoft/msbuild/issues/872 -->
+      <Copy SourceFiles="$(RepoRoot)/resources/MSBuild.exe.config"
+            DestinationFiles="$(SdkOutputDirectory)/MSBuild.exe.config" />
```

Comment:
Would it make more sense to put this file in the `publishOptions` in the src\dotnet\project.json file?  That way our project is a little more self-contained, and doesn't rely on this custom MSBuild task.  Just "dotnet publish" of `src\dotnet` will take care of copying this file to the output directory.